CHANGES.md

```diff
@@ -1,12 +1,17 @@
 wavesurfer.js changelog
 =======================
 
-6.5.0 (unreleased)
+6.5.0 (11.03.2023)
 ------------------
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
 - Respect mute state when changing volume during mute (#2502)
+- Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. #2691)
+- Chores:
+  - Docs: update the issue template and wavesurfer links (Docs: update the issue template and wavesurfer links #2671)
+  - Create FUNDING.yml (Create FUNDING.yml #2668)
```

Comment:
this is the type of useless changelog messages i don't like when automating things; please give it a meaningful description.